default.env

```diff
@@ -122,24 +122,21 @@ HLL_HOST=
 # The HLL RCON port (this is not the game or query port), ex: 27230
 HLL_PORT=
 
-# The RCON  password of your game server
+# The RCON password of your game server
 HLL_PASSWORD=
 
 # -----------------------------
 # RCON Web Setttings
 # -----------------------------
 
-# This is the port where the Rcon Website will be accessible. Feel free to change. 
-# If you're running multiple instances you have to change it per instance
-# If you want to put a reverse proxy on your rcon (e.g. for HTTPS) and block the direct 
-# access then use: 127.0.0.1:8010
```

Comment:
This is still possible, no?

Reply:
Maybe. Not sure, as I do not use reversing. + See the other note below.

Reply:
This is absolutely possible and we should keep that comment :)